tests/test_resources.py

```diff
@@ -1950,9 +1950,14 @@ def test_subscription_with_plan_ramp(self):
         self.assertEqual(len(new_sub.ramp_intervals), 2)
         self.assertEqual(new_sub.ramp_intervals[0].starting_billing_cycle, 1)
         self.assertEqual(new_sub.ramp_intervals[0].unit_amount_in_cents, '3000')
+        self.assertEqual(new_sub.ramp_intervals[0].starting_on, datetime(2023, 10, 12, 15, 38, 5, tzinfo=new_sub.ramp_intervals[0].starting_on.tzinfo))
+        self.assertEqual(new_sub.ramp_intervals[0].ending_on, datetime(2024, 1, 12, 15, 38, 5, tzinfo=new_sub.ramp_intervals[0].ending_on.tzinfo))
+
```

Comment:
We have tests that things serialize correctly, and this is a simple use case. I thought it didn't make sense to sprinkle these expectations everywhere, and one test case was sufficient(especially since the other test case that explicitly tests sub ramps use the same fixture